source/Node.js

```diff
@@ -91,7 +91,9 @@ function getPath ( node ) {
         if ( id = node.id ) {
             path += '#' + id;
         }
-        if ( className = node.className.trim() ) {
+
+
+        if (node.className && node.className.trim && (className = node.className.trim()) ) {
```

Comment:
It doesn't look like there's anything that actually doesn't have trim: http://kangax.github.io/compat-table/es5/#String.prototype.trim